data-files/samples/weapons.Experiment.Any

```diff
@@ -0,0 +1,151 @@
+{ 
+    description = "manytargets"; 
+    frameRate = 120; 
+    showHUD = true;
+    renderWeaponStatus = true;
+    cooldownMode = "ring";
+    cooldownColor = Color4(0.7,1.0,0.9,0.75);
```

Comment:
You could also do something like change cooldown color per session here as a demo of cascaded config if you'd like

Reply:
I'd suggest removing the cooldown indicator altogether for the autofire weapon sessions below

Reply:
I intentionally left it in for the autofire, but I took it out for the fastfire one. I think 200ms fire period is about the max speed at which a cooldown indicator makes sense, though it's definitely experiment specific.